### PR TITLE
CHK-101: Add clearer message for unavailable delivery items

### PR DESCRIPTION
**Pull request by darlenedms**
#### What is the purpose of this pull request?

Clarify to users which items are unavailable.

#### How should this be manually tested?

For all examples you need to proceed to Checkout to see the Omnishipping (on shipping step).
You can use any zip code you prefer, e.g.: `20740-580`

[*One item unavailable, pickup available* (pague menos)](https://unavailable--paguemenos.myvtex.com/checkout/cart/add/?sku=44953&qty=1&seller=1&sc=1&sku=54692&qty=1&seller=1&sc=1&sku=27451&qty=1&seller=1&sc=1&sku=50880&qty=1&seller=1&sc=1&sku=53017&qty=1&seller=1&sc=1&sku=6441&qty=1&seller=1&sc=1&sku=56456&qty=1&seller=1&sc=1&sku=52291&qty=1&seller=1&sc=1)
[*One item unavailable, pickup unavailable* (motorola br)](https://unavailable--brmotorola.myvtex.com/checkout/cart/add/?sku=133&qty=1&seller=1&sc=1&sku=10000372&qty=1&seller=1&sc=1&sku=93&qty=1&seller=1&sc=1)

omn_one_unavailable

[*Some unavailable*](https://unavailable2--vtexgame1.vtexcommercestable.com.br/checkout/cart/add?sku=31&qty=1&seller=1&sc=1&sku=334&qty=1&seller=1&sc=1&sku=285&qty=1&seller=1&sc=1&sku=308&qty=1&seller=1&sc=1&sku=291&qty=1&seller=1&sc=1&sku=296&qty=1&seller=1&sc=1&sku=307&qty=1&seller=1&sc=1&sku=331&qty=1&seller=1&sc=1&sku=289&qty=1&seller=1&sc=1&sku=286&qty=1&seller=1&sc=1)

omn_some_unavailable

[*All items unavailable*](https://unavailable2--vtexgame1.vtexcommercestable.com.br/checkout/cart/add/?sku=285&qty=1&seller=1&sc=1&sku=308&qty=1&seller=1&sc=1&sku=296&qty=1&seller=1&sc=1&sku=307&qty=1&seller=1&sc=1&sku=331&qty=1&seller=1&sc=1&sku=286&qty=1&seller=1&sc=1)

> If you accessed the "Some unavailable" link, clear your cart before accessing this one (they are the same workspace)

omn_all_unavailable

#### Screenshots or example usage

https://www.figma.com/file/GfG2o8GIgyCWMjNcpmjyrh/04.-Checkout-v6---Shipping---Split?node-id=14%3A2347

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/UnavailableDeliveryAvailablePickup.js [hunk 1]
```diff
@@ -1,73 +1,169 @@
...
-        className={`shp-alert-shipping-unavailable ${styles.alertPickup}`}
-        footer={
-          <div className={`shp-product-items ${styles.productItems}`}>
-            <ProductItems isAvailable={false} items={unavailableItems.items} />
+      <Fragment>
+        {orderFormItems.length === unavailableItems.items.length ? (
+          <Alert
+            className={`shp-alert-shipping-unavailable ${styles.alertPickup}`}
+          >
+            <p id="shp-unavailable-delivery-available-pickup">
```
Comment (augustocb) [inline react/components/UnavailableDeliveryAvailablePickup.js:53]: image

Can we turn that element into a `div` element? So we would avoid that extra bottom margin.

**Comment by augustocb** [comment]
@darlenedms Do we have some store without customisation to check the changes?

**Comment by augustocb** [comment]
The links for testing are not working for me :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Error due to accessing a `valid` prop of a `postalCode` that does not exist in scenarios where the pickup modal does not ask for the postal code.
 
+## [1.10.6-hkignore] - 2020-06-26
```
Comment (darlenedms) [inline CHANGELOG.md:36]: We need to drop the hkignore commit before merging.

**Pull request merged**